rllib/core/examples/simple_ppo_rl_module.py

```diff
@@ -0,0 +1,321 @@
+from dataclasses import dataclass, field
+import gym
+import torch
```

Comment:
Can we use our `try_import_torch()` utility here:

```
from ray.rllib.utils.framework import try_import torch

torch, nn = try_import_torch()
```

Reply:
It's an example script, so it won't break rllib entirely, but still, for consistency ...

Reply:
one thing I found annoying during development is that the returned torch actually does not let you control click to the torch module. I usually develop using actual `import torch` and then replace them at the end with `torch, _ = try_import_torch` which was should we opt for? Does try_import_torch give a better error than torch is not available? we only have either torch or tensorflow here anyway. It's not like both of them are available here.

Reply:
so if you agree with what I said, I should actually remove `try_import_torch` usages from this PR entirely since there is no file that needs both tf and torch in its import statements.

Reply:
Removing `try_torch_import` for now based on my reasoning above to move fwd with the merge. I'll change it if the offline discussions result in another consistent pattern.

Reply:
oh actually I take it back. :) the parent who imports `torch_rl_module` could still hit this issue when torch import fails even though we won't be using this class at run-time.